src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/questionWithResponses.tag

```diff
@@ -7,6 +7,8 @@
 <%@ attribute name="isShowRealQuestionNumber" type="java.lang.Boolean" required="true" %>
 <%@ attribute name="isSessionOpenForSubmission" type="java.lang.Boolean" required="true" %>
 
+<c:set var="isRecipientNameHidden" value="${questionWithResponses.question.recipientNameHidden}"/>
```

Comment:
Why is this the conditional used to determine whether the `Evaluee/Recipient` is displayed or not? What is the logic here?

Reply:
@wkurniawan07 
Without this conditional, `Evaluee/Recipient` was being displayed in Questions where there was no recipient name, like places like `Question 7` shown in the screenshot here : https://github.com/TEAMMATES/teammates/pull/8265#issuecomment-354286157, and other similar questions, and the UI did not look that much appealing for that reason.

Reply:
That is not the issue. My question is why the condition to be used to determine whether `Evaluee/Recipient` is displayed is `questionWithResponses.question.recipientNameHidden`. How is that condition representative of whether or not to display `Evaluee/Recipient`?

Reply:
There was a similar test previously used in `response.tag` file, 
```jsp
${isRecipientNameHidden ?  'style="display:none"' : 'style="text-align:right"'}>
```
and I noticed that the questions which should have `Evaluee/Recipient` have recipient names like this one : 
![screenshot 295](https://user-images.githubusercontent.com/10681755/35054344-35682a4a-fba4-11e7-97f2-c61f0f0836e1.png)

And the questions where the `Evaluee/Recipient` does not look appealing or appropriate does not have recipient names like this one : 
![screenshot 281](https://user-images.githubusercontent.com/10681755/34411591-90543cc0-ebfd-11e7-841b-353b7ac42725.png)

So, I used `recipientNameHidden` variable to test if the recipient name is hidden in that question or not, and based on that I displayed the word `Evaluee/Recipient`.